### PR TITLE
fix UNIX_TIMESTAMP NULL

### PR DESCRIPTION
**Pull request by yangwenmai**
when a column is datetime type, set Allow NULL，use UNIX_TIMESTAMP FUNC.

> Unkonwn args type for unix_timestamp 0

Then It will be OK.

### CODE REVIEW

File: expression/builtin_time.go [hunk 1]
```diff
@@ -1506,6 +1506,9 @@ func (b *builtinUnixTimestampSig) eval(row []types.Datum) (d types.Datum, err er
 		}
 	case types.KindMysqlTime:
 		t = args[0].GetMysqlTime()
+	case types.KindNull:
+		d.SetNull()
```
Comment (shenli) [inline expression/builtin_time.go]: It is not necessary to set d to null here. d is already null.

Comment (yangwenmai) [inline expression/builtin_time.go]: OK

**Comment by shenli** [comment]
LGTM
@XuHuaiyu PTAL

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**